tests/framework/artifacts.py

```diff
@@ -480,6 +480,11 @@ def rebase_snapshot(self, base):
         merge_memory_bitmaps(base.mem, self.mem)
         self._mem = base.mem
 
+    def delete_files(self):
```

Comment:
This could be called as part of the deconstructor to avoid explicit calls.

Reply:
Done.